### PR TITLE
Files Added in organization and sources

### PR DESCRIPTION
**Pull request by tapclicks-app**

### CODE REVIEW

File: organizations/yext_enhanced.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "id": "YEXT",
```
Comment (pfrisella) [inline organizations/yext_enhanced.json]: id should match file name. i.e. yext.json

File: sources/sizmek.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "id": "SIZMEK",
```
Comment (pfrisella) [inline sources/sizmek.json]: There are already 2 Sizmek sources (DSP, MDX), can you confirm neither of those is appropriate or that this source can't be more specific?

File: sources/twitter.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/twitter.json]: Perhaps this should be a bit more specific, Twitter Tweets?
if Twitter Social is more appropriate then I suggest renaming id and filename to TWITTER_SOCIAL

**Comment by tapclicks-app** [comment]
I signed it!

**Pull request merged**